src/sos/workflow_executor.py

```diff
@@ -1634,7 +1634,7 @@ def run_as_master(self, targets=None, mode=None) -> Dict[str, Any]:
                     time.sleep(0.1)
         except KeyboardInterrupt as e:
             if exec_error.errors:
-                failed_steps, pending_steps = dag.pending()
+                failed_steps, pending_steps = dag.pending()[1]
```

Comment:
This is not right....

Reply:
Sorry. I'll look back at your previous comment for this line.